src/main/java/model/PostRequest.java

```diff
@@ -1,6 +1,8 @@
 package model;
 
 public class PostRequest {
+  public static final String parameter = "/message?";
```

Comment:
constant parameters should be all Capital letters, you can name it PATH or REQUEST_PATH